docs/content/en/docs/reference/baremetal/bare-prereq.md

```diff
@@ -3,10 +3,78 @@ title: "Requirements for EKS Anywhere on Bare Metal"
 linkTitle: "Requirements"
 weight: 15
 description: >
-  Preparing a VMware vSphere provider for EKS Anywhere
+  Bare Metal provider requirements for EKS Anywhere
 ---
 
-ADD CONTENT TO DESCRIBE:
+To run EKS Anywhere on Bare Metal, you need the following:
 
-* Hardware requirements (IPMI, extra upgrade machines, etc.), including list of validated hardware models/families, etc.
-* Network requirements
+
+## Administrative machine
+
+Set up an Administrative machine as described in [Install EKS Anywhere ]({{< relref "../../getting-started/install/" >}}).
+
+## Computer requirements 
+
+The minimum number of physical computers needed to run EKS Anywhere in a non-production mode is:
+
+* Control plane physical machines: 1
+* Worker physical machines: 1
+
+The recommended number of physical machines for production is at least:
+
+* Control plane physical machines: 3
+* Worker physical machines: 2
+
+You will need an additional, temporary machine later when you go to upgrade a node.
+
+The computer hardware you need for your Bare Metal cluster must meet the following capacity requirements:
+
+* CPU: 2
+* Memory: 8GB RAM
+* Storage: 25GB
+
+## Network requirements
+
+Each machine must include the following features:
+
+* Network Interface Cards: At least one NIC is required. It must be capable of netbooting from iPXE. It must have at least 4GB of RAM for the OSIE/Hook boot and operation.
+* IPMI integration: An IPMI implementation (such a Dell Drac or HP iLO) on the computer's motherboard or on a separate expansion card. This feature is used to allow remote management of the machine, such as turning the machine on and off.
```

Comment:
iDRAC*

Reply:
Okay. I changed Drac to iDRAC. I also added a note saying that IPMI integration is recommended, but not required.